go.mod

```diff
@@ -3,80 +3,88 @@ module github.com/cybozu-go/cattage
 go 1.17
```

Comment:
I think this line needs to be updated to 1.18.

Reply:
It is not updated intentionally because we are not using the language features in the new version.